docs/source/daml/intro/13_Interfaces.rst

```diff
@@ -250,3 +250,8 @@ the resulting ``ContractId t`` into a ``ContractId IAsset`` using
   :language: daml
   :start-after: -- MK_ASSET_TEST_BEGIN
   :end-before: -- MK_ASSET_TEST_END
+
```

Comment:
I would remove this. We are restructuring this menu and I think that the next section will probably not be Next Steps in a very short space of time.

Also, our tone of voice in docs is more formal than here which is more of a blog-based tone of voice. Something to keep in mind maybe. We are planning to overhaul everything but it will take some time.